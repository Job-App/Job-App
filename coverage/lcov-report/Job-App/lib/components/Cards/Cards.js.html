
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for Job-App/lib/components/Cards/Cards.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../../index.html">All files</a> / <a href="index.html">Job-App/lib/components/Cards</a> Cards.js</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>0/15</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/6</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/1</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>0/1</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line low'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a></td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">var _interopRequireDefault=<span class="cstat-no" title="statement not covered" >require("@babel/runtime/helpers/interopRequireDefault");<span class="cstat-no" title="statement not covered" ></span>Object.defineProperty(exports,"__esModule",{value:true});<span class="cstat-no" title="statement not covered" >e</span>xports.default=void 0;v</span>ar _react=<span class="cstat-no" title="statement not covered" >_interopRequireDefault(require("react"));</span>var _reactNative=<span class="cstat-no" title="statement not covered" >require("react-native");</span>var _this=<span class="cstat-no" title="statement not covered" >this,</span>_jsxFileName=<span class="cstat-no" title="statement not covered" >"C:\\Users\\guo\\Desktop\\EECS393\\Job-App\\src\\components\\Cards\\Cards.js";</span>var _Dimensions$get=<span class="cstat-no" title="statement not covered" >_reactNative.Dimensions.get('screen'),</span>width=<span class="cstat-no" title="statement not covered" >_Dimensions$get.width,</span>height=<span class="cstat-no" title="statement not covered" >_Dimensions$get.height;</span>var Cards=<span class="cstat-no" title="statement not covered" >function <span class="fstat-no" title="function not covered" >Cards(</span>props){<span class="cstat-no" title="statement not covered" >return _react.default.createElement(_reactNative.TouchableOpacity,{activeOpacity:.8,__self:_this,__source:{fileName:_jsxFileName,lineNumber:14,columnNumber:9}},_react.default.createElement(_reactNative.View,{style:cardStyles.container,__self:_this,__source:{fileName:_jsxFileName,lineNumber:16,columnNumber:13}},_react.default.createElement(_reactNative.View,{style:cardStyles.companyContainer,__self:_this,__source:{fileName:_jsxFileName,lineNumber:17,columnNumber:17}},_react.default.createElement(_reactNative.Text,{style:cardStyles.jobText,__self:_this,__source:{fileName:_jsxFileName,lineNumber:18,columnNumber:21}},props.company||'N/A')),_react.default.createElement(_reactNative.View,{style:cardStyles.jobInfoContainer,__self:_this,__source:{fileName:_jsxFileName,lineNumber:19,columnNumber:17}},_react.default.createElement(_reactNative.Text,{style:cardStyles.text,__self:_this,__source:{fileName:_jsxFileName,lineNumber:20,columnNumber:21}},props.title||'N/A'),_react.default.createElement(_reactNative.Text,{style:cardStyles.text,__self:_this,__source:{fileName:_jsxFileName,lineNumber:21,columnNumber:21}},props.closing||'N/A'))));}</span>;</span>var cardStyles=<span class="cstat-no" title="statement not covered" >_reactNative.StyleSheet.create({container:{width:width-96,alignSelf:'center',marginTop:12,height:80,backgroundColor:'#CCCCCC',flexDirection:'row'},text:{flex:1,textAlignVertical:'center',marginLeft:12},jobText:{margin:12,fontSize:24,flex:1},companyContainer:{alignItems:'flex-start',flex:1.5,margin:12,marginRight:6},jobInfoContainer:{alignItems:'flex-start',flex:2,margin:12,marginLeft:6,flexDirection:'column'}});</span>var _default=<span class="cstat-no" title="statement not covered" >Cards;<span class="cstat-no" title="statement not covered" ></span>exports.default=_default;</span></pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Thu Oct 22 2020 05:31:36 GMT-0400 (Eastern Daylight Time)
            </div>
        </div>
        <script src="../../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../../sorter.js"></script>
        <script src="../../../../block-navigation.js"></script>
    </body>
</html>
    