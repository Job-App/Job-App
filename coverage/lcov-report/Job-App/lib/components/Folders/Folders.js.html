
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for Job-App/lib/components/Folders/Folders.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../../index.html">All files</a> / <a href="index.html">Job-App/lib/components/Folders</a> Folders.js</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>0/20</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/3</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>0/1</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line low'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a></td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">var _interopRequireDefault=<span class="cstat-no" title="statement not covered" >require("@babel/runtime/helpers/interopRequireDefault");<span class="cstat-no" title="statement not covered" ></span>Object.defineProperty(exports,"__esModule",{value:true});<span class="cstat-no" title="statement not covered" >e</span>xports.default=void 0;v</span>ar _react=<span class="cstat-no" title="statement not covered" >_interopRequireDefault(require("react"));</span>var _reactNative=<span class="cstat-no" title="statement not covered" >require("react-native");</span>var _Cards=<span class="cstat-no" title="statement not covered" >_interopRequireDefault(require("./../Cards/Cards"));</span>var _this=<span class="cstat-no" title="statement not covered" >this,</span>_jsxFileName=<span class="cstat-no" title="statement not covered" >"C:\\Users\\guo\\Desktop\\EECS393\\Job-App\\src\\components\\Folders\\Folders.js";</span>var _Dimensions$get=<span class="cstat-no" title="statement not covered" >_reactNative.Dimensions.get('screen'),</span>width=<span class="cstat-no" title="statement not covered" >_Dimensions$get.width,</span>height=<span class="cstat-no" title="statement not covered" >_Dimensions$get.height;</span>var mapCards=<span class="cstat-no" title="statement not covered" >function <span class="fstat-no" title="function not covered" >mapCards(</span>){var arr=<span class="cstat-no" title="statement not covered" >[{id:1,company:"Apple",title:"Software Engineer",closing:"10/22/2020"},{id:2,company:"Google",title:"SWE",closing:"10/23/2020"},{id:3,company:"Netflix",title:"SDE",closing:"10/24/2020"}];<span class="cstat-no" title="statement not covered" ></span>return arr.map(<span class="fstat-no" title="function not covered" >fu</span>nction(item){<span class="cstat-no" title="statement not covered" >return _react.default.createElement(_Cards.default,{id:item.id,company:item.company,title:item.title,closing:item.closing,__self:_this,__source:{fileName:_jsxFileName,lineNumber:33,columnNumber:13}});}</span>);}</span>;</span>var Folders=<span class="cstat-no" title="statement not covered" >function <span class="fstat-no" title="function not covered" >Folders(</span>props){<span class="cstat-no" title="statement not covered" >return _react.default.createElement(_reactNative.View,{style:styles.container,__self:_this,__source:{fileName:_jsxFileName,lineNumber:44,columnNumber:9}},_react.default.createElement(_reactNative.Text,{style:[styles.text,styles.title],__self:_this,__source:{fileName:_jsxFileName,lineNumber:45,columnNumber:13}},props.name),mapCards());}</span>;</span>var styles=<span class="cstat-no" title="statement not covered" >_reactNative.StyleSheet.create({container:{width:width-48,alignSelf:'center',marginTop:24,borderWidth:1,paddingBottom:24},text:{margin:12,marginBottom:0},title:{fontSize:36,fontWeight:'bold'}});</span>var _default=<span class="cstat-no" title="statement not covered" >Folders;<span class="cstat-no" title="statement not covered" ></span>exports.default=_default;</span></pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Thu Oct 22 2020 05:31:36 GMT-0400 (Eastern Daylight Time)
            </div>
        </div>
        <script src="../../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../../sorter.js"></script>
        <script src="../../../../block-navigation.js"></script>
    </body>
</html>
    