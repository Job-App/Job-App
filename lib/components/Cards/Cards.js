var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");Object.defineProperty(exports,"__esModule",{value:true});exports.default=void 0;var _react=_interopRequireDefault(require("react"));var _reactNative=require("react-native");var _this=this,_jsxFileName="C:\\Users\\guo\\Desktop\\EECS393\\Job-App\\src\\components\\Cards\\Cards.js";var _Dimensions$get=_reactNative.Dimensions.get('screen'),width=_Dimensions$get.width,height=_Dimensions$get.height;var Cards=function Cards(props){return _react.default.createElement(_reactNative.TouchableOpacity,{activeOpacity:.8,__self:_this,__source:{fileName:_jsxFileName,lineNumber:14,columnNumber:9}},_react.default.createElement(_reactNative.View,{style:cardStyles.container,__self:_this,__source:{fileName:_jsxFileName,lineNumber:16,columnNumber:13}},_react.default.createElement(_reactNative.View,{style:cardStyles.companyContainer,__self:_this,__source:{fileName:_jsxFileName,lineNumber:17,columnNumber:17}},_react.default.createElement(_reactNative.Text,{style:cardStyles.jobText,__self:_this,__source:{fileName:_jsxFileName,lineNumber:18,columnNumber:21}},props.company||'N/A')),_react.default.createElement(_reactNative.View,{style:cardStyles.jobInfoContainer,__self:_this,__source:{fileName:_jsxFileName,lineNumber:19,columnNumber:17}},_react.default.createElement(_reactNative.Text,{style:cardStyles.text,__self:_this,__source:{fileName:_jsxFileName,lineNumber:20,columnNumber:21}},props.title||'N/A'),_react.default.createElement(_reactNative.Text,{style:cardStyles.text,__self:_this,__source:{fileName:_jsxFileName,lineNumber:21,columnNumber:21}},props.closing||'N/A'))));};var cardStyles=_reactNative.StyleSheet.create({container:{width:width-96,alignSelf:'center',marginTop:12,height:80,backgroundColor:'#CCCCCC',flexDirection:'row'},text:{flex:1,textAlignVertical:'center',marginLeft:12},jobText:{margin:12,fontSize:24,flex:1},companyContainer:{alignItems:'flex-start',flex:1.5,margin:12,marginRight:6},jobInfoContainer:{alignItems:'flex-start',flex:2,margin:12,marginLeft:6,flexDirection:'column'}});var _default=Cards;exports.default=_default;