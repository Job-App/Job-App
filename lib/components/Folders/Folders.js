var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");Object.defineProperty(exports,"__esModule",{value:true});exports.default=void 0;var _react=_interopRequireDefault(require("react"));var _reactNative=require("react-native");var _Cards=_interopRequireDefault(require("./../Cards/Cards"));var _this=this,_jsxFileName="C:\\Users\\guo\\Desktop\\EECS393\\Job-App\\src\\components\\Folders\\Folders.js";var _Dimensions$get=_reactNative.Dimensions.get('screen'),width=_Dimensions$get.width,height=_Dimensions$get.height;var mapCards=function mapCards(){var arr=[{id:1,company:"Apple",title:"Software Engineer",closing:"10/22/2020"},{id:2,company:"Google",title:"SWE",closing:"10/23/2020"},{id:3,company:"Netflix",title:"SDE",closing:"10/24/2020"}];return arr.map(function(item){return _react.default.createElement(_Cards.default,{id:item.id,company:item.company,title:item.title,closing:item.closing,__self:_this,__source:{fileName:_jsxFileName,lineNumber:33,columnNumber:13}});});};var Folders=function Folders(props){return _react.default.createElement(_reactNative.View,{style:styles.container,__self:_this,__source:{fileName:_jsxFileName,lineNumber:44,columnNumber:9}},_react.default.createElement(_reactNative.Text,{style:[styles.text,styles.title],__self:_this,__source:{fileName:_jsxFileName,lineNumber:45,columnNumber:13}},props.name),mapCards());};var styles=_reactNative.StyleSheet.create({container:{width:width-48,alignSelf:'center',marginTop:24,borderWidth:1,paddingBottom:24},text:{margin:12,marginBottom:0},title:{fontSize:36,fontWeight:'bold'}});var _default=Folders;exports.default=_default;