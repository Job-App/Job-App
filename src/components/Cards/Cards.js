import React from 'react';
import {
  View,
  Text,
  TouchableOpacity,
  StyleSheet,
  Dimensions,
} from 'react-native';

const {width, height} = Dimensions.get('screen');

const Cards = (props) => {
    return(
        <TouchableOpacity
            activeOpacity={.8}>
            <View style={cardStyles.container}>
                <View style={cardStyles.companyContainer}>
                    <Text style={cardStyles.jobText}>{props.company || 'N/A'}</Text></View>
                <View style={cardStyles.jobInfoContainer}>
                    <Text style={cardStyles.text}>{props.title || 'N/A'}</Text>
                    <Text style={cardStyles.text}>{props.closing || 'N/A'}</Text>
                </View>
            </View>
        </TouchableOpacity>
    )
}

const cardStyles = StyleSheet.create({
    container: {
        width: width - 48,
        alignSelf: 'center',
        marginTop: 12,
        height: 80,
        backgroundColor: '#CCCCCC',
        flexDirection: 'row'
    },
    text: {
        flex: 1,
        textAlignVertical: 'center',
        marginLeft: 12,
    },
    jobText: {
        margin: 12,
        fontSize: 24,
        flex: 1
    },
    companyContainer: {
        alignItems: 'flex-start',
        flex: 1.5,
        margin: 12,
        marginRight: 6,
    },
    jobInfoContainer: {
        alignItems: 'flex-start',
        flex: 2,
        margin: 12,
        marginLeft: 6,
        flexDirection: 'column'
    }
});

export default Cards;