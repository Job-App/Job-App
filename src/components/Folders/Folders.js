import React from 'react';
import {
  View,
  Text,
  TouchableOpacity,
  StyleSheet,
  Dimensions,
} from 'react-native';

import Cards from './../Cards/Cards';

const {width, height} = Dimensions.get('screen');

const Folders = (props) => {
    return(
        <View style={styles.container}>
            <Text style={[styles.text,styles.title]}>{props.name}</Text>
            <Cards />
        </View>
    )
}

const styles = StyleSheet.create({
    container: {
        width: width - 48,
        alignSelf: 'center',
        marginTop: 24,
        borderWidth: 1,
        paddingBottom: 24
    },
    text: {
        margin: 12,
        marginBottom: 0
    },
    title: {
        fontSize: 36,
        fontWeight: 'bold'
    }
});

export default Folders;