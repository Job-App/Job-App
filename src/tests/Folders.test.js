import React from 'react';
import {shallow} from 'enzyme';
import Folders from '../components/Folders/Folders.js';
import Cards from '../components/Cards/Cards.js';

const mockProps = {
  navigation: {navigate: jest.fn()},
  name: "Active"
};

test('Testing Folders component', () => {
  const wrapper = shallow(<Cards {...mockProps} />);
  expect(wrapper).toMatchSnapshot();
});

const mockItem = {
  key: 3,
  id: 3,
  company: "Company",
  title: "Title",
  closing: "10/31/2020"
};

test('Testing listItemView function', () => {
  const listItem = jest.fn();
  listItem.mockReturnValue('listItemView invoked');
  const wrapper = shallow(<Cards {...mockItem} />);
  expect(listItem.mock.calls.length(wrapper)).toBe(1);
})
