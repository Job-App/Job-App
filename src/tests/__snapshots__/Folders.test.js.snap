// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Folders component 1`] = `
<View
  style={94}
>
  <Text
    style={
      Array [
        95,
        96,
      ]
    }
  />
  <Cards
    closing="10/22/2020"
    company="Apple"
    id={1}
    title="Software Engineer"
  />
  <Cards
    closing="10/23/2020"
    company="Google"
    id={2}
    title="SWE"
  />
  <Cards
    closing="10/24/2020"
    company="Netflix"
    id={3}
    title="SDE"
  />
</View>
`;
